webui/react/src/pages/FlatRuns/columns.ts

```diff
@@ -51,10 +51,20 @@ export const runColumns = [
   'parentArchived',
 ] as const;
 
+const EXCLUDED_SEARCH_DEFAULT_COLUMNS: RunColumn[] = [
+  'searcherType',
+  'searcherMetric',
+  'searcherMetricsVal',
+];
+
 export type RunColumn = (typeof runColumns)[number];
 
 export const defaultRunColumns: RunColumn[] = [...runColumns];
 
+export const defaultSearchRunColumns: RunColumn[] = [...runColumns].filter(
```

Comment:
nit: since you're filtering the array (which returns a new array) I don't think the destructuring is necessary